cpg_workflows/defaults.toml

```diff
@@ -1,4 +1,7 @@
 [workflow]
+# Cohorts to use as inputs.
+# input_cohorts = []
+
 # Datasets to load inputs. If not provided, datasets will be determined
 # automatically based on the input provider implementation:
 #input_datasets = []
```

Comment:
does cohorts-as-input negate the need for input_datasets, or is there a transitional period where both are accepted? I remember some discussion on this, but not sure where we ended up

Reply:
Same comment applies for this whole list of arguments; skip_sgs, only_sgs, etc.

Reply:
I just saw the deprecation message below, so I guess for a while we're swapping input_datasets -> dataset_names behind the scenes.

Reply:
We will be supporting both for an interim period while we make the transition across :)